.github/workflows/release_canary.yml

```diff
@@ -11,6 +11,10 @@ on:
         type: number
         required: false
         default: 16
+      registry:
+        type: string
+        required: false
+        default: registry.npmjs.org
```

Comment:
I think a string input is prone to error. Given that we only want to publish to either npm _or_ GPR (with the former used more frequently than the latter), could you use a `boolean`  type instead and bring this logic inside? Something like `useGPR`, which defaults to `false`. 

Then in the step...

```
if [ "$useGPR" = "true" ]; then
  registry="//npm.pkg.github.com/"
else
  registry="//registry.npmjs.org/"
fi
cat << EOF > "$HOME/.npmrc"
  $registry:_authToken=$NPM_TOKEN
EOF
```

Reply:
This is a really good point. I just pushed the update including this change.